### DISCUSSION TITLE
Make maintiners comments more noticeable

### DISCUSSION THREAD

**Discussion by RosterIn** [community/community#31916] [Projects and Issues]
Currently in both Issues and Pull requests there are indications of who the user is: first time contributor, contributor, member
These are not very noticeable and even if are not everyone understand the difference.
This makes people to take suggestions from "untrusted" users rather than trusted.

Please make the member role more noticeable and clear.

**Comment by jfullstackdev** [comment]
is the tag `Maintainer` and `Collaborator` not enough ?

**Comment by RosterIn** [comment]
@jdevstatic 
The tag is not maintainer it's member (which is not so clear)
And no.. it's not enough because it's not noticeable. People don't understand who has write access and who is just a random person who comments.. they often tag wrong people as result and take wrong actions.

You need to make it very noticeable to everyone who is "offical" and who is not.
Take this exact case of this discussion. I have no way of knowing that you are an official GitHub supporter.

In forums (any forum that you will go online) it's always so clear who is a moderator and what are the ranks between them.. in GitHub it's very not clear.

People tag me and ask me to merge thier PRs in repos that I have no write access to.. they do that because I helped with review yet they are confused and think that I am a member of that repo while I am not.

**Comment by jfullstackdev** [comment]
having `Maintainer` and `Collaborator` tags in this Discussion means they are official, other than that, if you hover on their names, you will see an official `Staff` like this 

![image](https://user-images.githubusercontent.com/47092464/189488643-e8206e33-c9ea-44db-8e96-bdba9740893d.png)

but it's not always the case, some don't have the Staff badge on their profile at all, maybe it's in terms of their jobs, who will receive the badge or not

maybe a Staff here will explain it further, 

and if you create your own Discussion at your own Repo, you will have that `Maintainer` tag too, but don't be confused, it only means you are the maintainer of your own repo but not here on GitHub official repo

and this is the official documentation for them

https://github.com/orgs/community/discussions/28776

you can also ask them questions there for more visibility
